### PR TITLE
[DOC] add doc for file.rename method

### PR DESCRIPTION
**Pull request by Remi-Gau**

### CODE REVIEW

File: +bids/File.m [hunk 1]
```diff
@@ -386,6 +389,73 @@
...
+      %   %% Get a specific file from a dataset to rename
+      %
+      %   BIDS = bids.layout(path_to_dataset)
+      %
+      %   % construct a filter to get only the file we want/
+      %   subject = '001';
+      %   run = '001';
+      %   suffix = 'bold';
+      %   task = 'faceRep';
+      %   filter = struct('sub', subject, 'task', task, 'run', run, 'task, task');
```
Comment (CerenB) [inline +bids/File.m:443]: there's a small typo? `'task,task'` ?

Comment (Remi-Gau) [reply +bids/File.m:443]: yup 
thanks

Comment (Remi-Gau) [inline +bids/File.m:443]:
```suggestion
      %   filter = struct('sub', subject, 'task', task, 'run', run, 'suffix', suffix);
```

**Comment by Remi-Gau** [comment]
@all-contributors please add @CerenB  for review

**Pull request merged**